packages/website/src/building-blocs/PageLayout.tsx

```diff
@@ -21,6 +21,7 @@ interface PlaygroundProps {
     title: string;
     code: string;
     layout?: 'horizontal' | 'vertical';
+    exampleLayout?: 'horizontal' | 'vertical';
```

Comment:
I believe this is misleading and unnecessary because PageLayoutProps extends this interface. So you could end-up with something very confusing like

```tsx
<PageLayout 
    layout='vertical' 
    exampleLayout='horizontal'
    examples={{
        whatever: {code: whatever, layout: 'horizontal', exampleLayout: 'vertical'},
    }}
/>
```

```suggestion
```